app/src/main/java/org/simple/clinic/editpatient/EditPatientEvent.kt

```diff
@@ -70,3 +70,5 @@ data class AlternativeIdChanged(val alternativeId: String) : EditPatientEvent()
 }
 
 data class InputFieldsLoaded(val inputFields: InputFields) : EditPatientEvent()
+
+data class ColonyOrVillagesFetched(val colonyOrVillages: List<String>?) : EditPatientEvent()
```

Comment:
The `List` will never be `null`. We can make the `getColonyOrVillages` as non-null. If there is no data in the db, Room database will return an empty list instead of a nullable object for lists. So we don't have to handle nullable here.